jsonschema/cli.py

```diff
@@ -69,19 +70,49 @@ def validation_success(self, **kwargs):
 
 @attr.s
 class _PrettyFormatter(object):
+    _MESSAGE_BAR_CHAR = '═'
```

Comment:
Minor: style-wise `jsonschema` uses double quotes (unless there's a double quote in the string, then single quotes).

Reply:
Ah, sorry.  I hadn't noticed.  My personal habit is to use single quotes.  I'll update that.